server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -352,14 +452,35 @@ public RemoteClusterStateManifestInfo writeIncrementalMetadata(
             indicesRoutingToUpload.size()
         );
         if (durationMillis >= slowWriteLoggingThreshold.getMillis()) {
+            // TODO update logs to add more details about objects uploaded
             logger.warn(
-                "{} which is above the warn threshold of [{}]; {}",
-                clusterStateUploadTimeMessage,
+                "writing cluster state took [{}ms] which is above the warn threshold of [{}]; "
+                    + "wrote  metadata for [{}] indices and skipped [{}] unchanged indices, coordination metadata updated : [{}], "
+                    + "settings metadata updated : [{}], templates metadata updated : [{}], custom metadata updated : [{}]",
+                durationMillis,
                 slowWriteLoggingThreshold,
-                metadataUpdateMessage
+                numIndicesUpdated,
+                numIndicesUnchanged,
+                updateCoordinationMetadata,
+                updateSettingsMetadata,
+                updateTemplatesMetadata,
+                customsToUpload.size()
             );
         } else {
             logger.info("{}; {}", clusterStateUploadTimeMessage, metadataUpdateMessage);
+            logger.info(
```

Comment:
Can we please change the log level to trace or debug? This is leading to lot of logs while running a cluster on account of remote cluster state uploads.

Reply:
Raised PR #14814 for this